<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDesignation</class>
 <widget class="QWidget" name="addDesignation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADD DESIGNATION</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QComboBox" name="DesignationList"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="searchBox">
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Designation Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="designationText">
         <property name="maxLength">
          <number>15</number>
         </property>
         <property name="placeholderText">
          <string>ENTER DESIGNATION NAME</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>       Salary          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="salaryText">
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="placeholderText">
        <string>ENTER SALARY ACCORDING DESIGNATION</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
